pkg/common/common.go

```diff
@@ -0,0 +1,14 @@
+package common
```

Comment:
From the Go Blog on [Package Names](https://go.dev/blog/package-names): "Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains".

I know we have a `util` package under `cli`. That should change too at some point in time, but we should avoid creating new ones.